api/custom_auth/jwt_cookie/views.py

```diff
@@ -0,0 +1,11 @@
+from djoser.views import TokenDestroyView
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework_simplejwt.tokens import SlidingToken
+
+
+class TokenDestroyJWTSlidingBlacklistView(TokenDestroyView):
+    def post(self, request: Request) -> Response:
+        if isinstance(request.auth, SlidingToken):
+            request.auth.blacklist()
+        return super().post(request)
```

Comment:
I'm interested to know when this endpoint should be called? It feels like this should just be done on the generic logout endpoint?

Reply:
>  the generic logout endpoint?

Which is...?

Reply:
I guess the real question here is why am I adding support for classic token deletion on logout. The answer is "no idea, it shouldn't be there."

I'm modifying it to only invalidate the JWT if it's present.

Reply:
This is done.

Reply:
Sorry, I think I misunderstood this code before. I didn't really notice that we were subclassing the `TokenDestroyView`. I guess this now relates to the question that I've asked [here](https://github.com/Flagsmith/flagsmith/pull/4662#discussion_r1777540740) in my second review.

Reply:
Yeah I got misled by your comment as well. Will revert tomorrow.